inventory/packages/apt_deb.go

```diff
@@ -31,11 +31,15 @@ var (
 	dpkgquery string
 	aptGet    string
 
-	dpkgQueryArgs        = []string{"-W", "-f", "${Package} ${Architecture} ${Version}\n"}
-	aptGetInstallArgs    = []string{"install", "-y"}
-	aptGetRemoveArgs     = []string{"remove", "-y"}
-	aptGetUpdateArgs     = []string{"update"}
-	aptGetUpgradableArgs = []string{"full-upgrade", "--just-print", "-V"}
+	dpkgQueryArgs     = []string{"-W", "-f", "${Package} ${Architecture} ${Version}\n"}
+	aptGetInstallArgs = []string{"install", "-y"}
+	aptGetRemoveArgs  = []string{"remove", "-y"}
+	aptGetUpdateArgs  = []string{"update"}
+
+	aptGetUpgradeCmd     = "upgrade"
+	aptGetFullUpgradeCmd = "full-upgrade"
+	aptGetDistUpgradeCmd = "dist-upgrade"
+	aptGetUpgradableArgs = []string{"--just-print", "-qq"}
```

Comment:
does it make sense to have a separate package for "package-managers"? currently if patch package has to use anything from package managers, it has to import inventory which does not sense.

Reply:
Yeah, I plan on moving the directory eventually. Right now I'm still migration all package calls to this library.